templates/environment/versions/cf-v1.3.0.yml

```diff
@@ -21,6 +22,9 @@ Parameters:
   AppDNSDelegationRole:
     Type: String
     Default: ""
+  NATWorkloads:
```

Comment:
nit: can we add this parameter below "EFSWorkloads" to just keep these env controller params together.